src/karts/kart.cpp

```diff
@@ -370,6 +376,12 @@ void Kart::reset()
     m_has_caught_nolok_bubblegum = false;
     m_is_jumping           = false;
 
+    for (int i=0;i<30;i++)
+    {
+        m_previous_xyz[i] = getXYZ();
+    }
+    time_previous_counter = 0.0f;
+
```

Comment:
The 30 appears to be dependent on the FPS (there is a 0.25 mentioned below). Use the stk_config functions!